seatunnel-e2e/seatunnel-engine-e2e/connector-seatunnel-e2e-base/src/test/resources/seatunnel.yaml

```diff
@@ -31,3 +31,11 @@ seatunnel:
         max-retained: 3
         plugin-config:
           namespace: /tmp/seatunnel/checkpoint_snapshot/
+    telemetry:
+      metric:
+         enabled: false
+      logs:
+         scheduled-deletion-enable: true
+         cron: "* * * * *"
+         keep-time: 30000
```

Comment:
The default value is different from TELEMETRY_LOGS_SCHEDULED_DELETION_KEEP_TIME.

Reply:
Test case, same cannot be triggered

Reply:
You can use other methods to circumvent this problem. Otherwise, seatunnel will clear the logs in a short time by default, which may cause trouble in troubleshooting problems based on logs.

Reply:
This case in e2e. Do we need to consider this?

Reply:
I see. Thank you.